mifospay/src/main/java/org/mifos/mobilewallet/mifospay/kyc/ui/KYCDescriptionFragment.java

```diff
@@ -95,20 +133,32 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
 
     @OnClick(R.id.cv_lvl1)
     public void onLevel1Clicked() {
-        replaceFragmentUsingFragmentManager(KYCLevel1Fragment.newInstance(), true,
-                R.id.container);
+        KYCLevel1Fragment kycLevel1Fragment=new KYCLevel1Fragment();
```

Comment:
So many

Reply:
@letelete  I think this code had to be changed for the kyc section to run properly kindly run and see it for yourself

Reply:
You could make use of multiple on click assignment for the buttons (`@OnClick( { R.id.1, R.id.2, R.id.3 } )`) and then using a method with `View clickedButton` argument you could handle the view id in the switch to decide which fragment should've been opened.